zerver/lib/markdown/fenced_code.py

```diff
@@ -189,6 +193,30 @@ def generic_handler(
         return TexHandler(processor, output, fence)
     elif lang == "spoiler":
         return SpoilerHandler(processor, output, fence, header)
+    elif lang in (
+        "blockdiag",
+        "seqdiag",
+        "nwdiag",
+        "actdiag",
+        "mermaid",
+        "plantuml",
+        "svgbob",
+        "packetdiag",
+        "rackdiag",
+        "graphviz",
+        "erd",
+        "excalidraw",
+        "vega",
+        "vegalite",
+        "ditaa",
+        "nomnoml",
+        "bpmn",
+        "bytefield",
+        "wavedrom",
+        "c4plantuml",
+        "umlet",
+    ):
+        return KrokiHandler(processor, output, fence, lang)
```

Comment:
This is exactly what I [warned](https://chat.zulip.org/#narrow/stream/49-development-help/topic/pull.20docker.20images.20and.20run.20in.20zulip.20vagrant.20file/near/1129292) we should avoid: this steals part of the Pygments namespace for Kroki, and makes it impossible to access the Pygments syntax highlighter for `graphviz`, or for other languages in this list that Pygments might add in the future.

Reply:
Fixed this issue in the next commit